### PR TITLE
Gh actions test

### PR DESCRIPTION
**Pull request by solderq35**
I set up github pages and s3 gh actions workflow like energy-dashboard, but I get a weird cors error on the console, and I can't seem to login / view historical data.

I got deployment errors with node10 so I switched to node12 (which at least seems ok in development), and I fixed some linter errors also. Also tried switching the bucket file path in packaged.yml. Didn't fix cors issue.

### CODE REVIEW

File: packaged.yaml [hunk 1]
```diff
@@ -69,9 +69,9 @@ Resources:
...
       Policies:
       - AmazonDynamoDBFullAccess
-      CodeUri: s3://osu-so-serverless-builds/f1dfd6a4863b36f25384e92ae6eabb59
+      CodeUri: s3://osu-so-serverless-builds/3df2e338874ddf59b215bd09b8ae6bbc
       Handler: app.questions
-      Runtime: nodejs10.x
+      Runtime: nodejs12.x
       Layers:
       - Ref: LambdaCommonLayer
       Events:
```
Comment (MilanDonhowe) [inline packaged.yaml:77]: We should probably delete this file, I think it's leftover from an old build command.

File: backend/template.yaml [hunk 2]
```diff
@@ -73,7 +73,7 @@ Resources:
         - AmazonDynamoDBFullAccess
       CodeUri: app/carbon
       Handler: app.questions
-      Runtime: nodejs10.x
+      Runtime: nodejs12.x
       Layers:
         - !Ref LambdaCommonLayer
       Events:
```
Comment (MilanDonhowe) [inline backend/template.yaml:79]: We should probably update the lambda common layer reference

**Review by MilanDonhowe — CHANGES_REQUESTED** [review]
Looking good so far, few changes we'll want to make before merging:

- [ ] delete packaged.yaml
- [ ] update common layer reference (should match dashboard's reference in its template.yaml)

And after that I think we should be good to merge the CI changes

**Comment by solderq35** [comment]
I couldn't figure out how to fix the login issues but I removed the packaged.yaml, updated lambda layer reference, and added vue store debugging.

**Pull request merged**